openstack-upi/openstack-upi.md

```diff
@@ -0,0 +1,317 @@
+---
+title: openstack-upi
+authors:
+  - "@tomassedovic"
+reviewers:
+  - "@mandre"
+  - "@luis5tb"
+  - "@wking"
+  - "@cuppett"
+approvers:
+  - TBD
+creation-date: 2019-09-10
+last-updated: 2019-12-03
+status: implementable
+---
+
+# OpenStack UPI
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+The initial OpenStack support for OpenShift 4 was around the IPI
+workflow. While this is the most convenient approach for the end-users
+of OpenStack, there are situations where it falls short. People who
+want to use OpenStack UPI today have to fall back on
```

Comment:
> People who want to use OpenStack UPI today have to fall back on
the [bare metal][baremetal-upi] document or try to translate concepts
from the AWS one.

hmm, not sure what you mean here?